CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contribution guidelines
+
++ [Before adding a recipe](#before-adding-a-recipe)
+  + [Labels](#labels)
++ [Recipe Structure](#recipe-structure)
+  + [Badges](#badges)
++ [Pull requests](pull-requests)
++ [Instructions for maintainers](#instructions-for-maintainers)
+
+## Before adding/editing a recipe
+
+ 1. Search this repo's [issues](https://www.github.com/foundersandcoders/nazareth-recipes/issues) to see if an issue exists for the recipe.
+ 2. If the issue does not exist, create it.
+ 3. If your new issue relates to any others, reference those issues in the body. This enables others to follow the history of the topic.
+ 4. Assign [the appropriate label](#labels). **All recipe issues must have labels**
+ 5. Once you are sure of what you need to do and that it is needed, assign yourself to the issue.
+ 6. Clone, and create a new branch for your work
+
+### Labels
+Advice for labeling your recipes:
+
+  - Indicate what meal your recipe might apply to (breakfast, lunch or dinner) - it can apply to more than one.
+  - Indicate whether or not the meal is vegetarian friendly
+  - Add any other labels you feel might be appropriate.
+
+## Recipe Structure
+Use the following skeleton guide
+```
+# Name of Dish Here
+
+- Badges here
+
+**Head Chef:** 
+
+**Sous Chef:** 
+
+**Date Cooked:** 
+
+**Author of Recipe:** 
+
+## Ingredients
+- Add all the ingredients here
+
+## Preparation
+
+1. Add step by step instructions here, ideally including photos of the process and/or finished product
+2. etc
+
+Photos of meal participants enjoying the food should come after the recipe
+```
+### Badges
+
+All recipes should have the following badges: [Difficulty](#difficulty), [Speed](#speed), [Reception](#reception), [Cost](#cost), [Vegetarian](#vegetarian), [Vegan](#vegan)
+
+Feel free to add other any badges you deem appropriate!
+
+The badges can be contructed using the following url: ```https://img.shields.io/badge/<SUBJECT>-<STATUS>-<COLOUR>.svg``` (credit: https://shields.io/)
+
+The subject should be one of the above, see directions below for status and colour guidance.
+#### Difficulty ![Difficulty Badge](https://img.shields.io/badge/Difficulty-70%25-orange.svg)
+- STATUS: A percentage. Format should be the percentage number followed by ```%25```
+- COLOUR: One of 5 colours, corresponding to the status as follows:
+  - 0-19% -```brightgreen```
+  - 20-39% -```green```
+  - 40-59% -```yellow```
+  - 60-79% -```orange```
+  - 80-100% -```red```
+
+#### Speed ![Speed Badge](https://img.shields.io/badge/Speed-1hr-yellow.svg)
+- STATUS: A time, in hours: ```0.5hrs, 1hr, 1.5hrs```
+- COLOUR: One of 5 colours, corresponding to the status as follows:
+  - STATUS < 0.5 -```brightgreen```
+  - 0.5< STATUS <1 -```green```
+  - 1< STATUS <1.5 -```yellow```
+  - 1.5< STATUS <2 -```orange```
+  - 2< STATUS -```red```
```

Comment:
Why 'STATUS' 5 times and not just the number ranges like the above label instructions?